Hello Microverse

danidreis opened the pull request:
These files were created as a result of the "Set up a "Hello Microverse" project".

danidreis committed:
Create .gitignore

danidreis committed:
Create workflows

danidreis committed:
Unsucessfull atempt to create folder

danidreis committed:
Create linters

danidreis committed:
Create Readme file

danidreis committed:
Create html file

danidreis committed:
Create css file

danidreis committed:
Create test file

danidreis committed:
Update test.md

danidreis committed:
Add viewport tag

Fix for  "Error    A 'viewport' meta element was not specified.  meta-viewport"

ShadyShawkat commented on README.md:
```diff
@@ -1 +1,27 @@
+
+## Author
+
+👤 **Daniela Reis**
+
+
```
- [x] Kindly add your social links (or what you have of them) to your `README` file (GitHub, LinkedIn, Twitter).

ShadyShawkat commented on README.md:
```diff
@@ -1 +1,27 @@
+- HTML/CSS
+
+
+## Getting Started
+
+**This is the first repository created for the Microverse**
+
+This file was made for review.
```
- [x] Please edit this section to contain information on how to set up your project locally.

ShadyShawkat commented on README.md:
```diff
@@ -1 +1,27 @@
-# MicroverseStudent
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> "Hello Microverse" project
+
+
+## Built With
+
+- HTML/CSS
+
+
+## Getting Started
+
+**This is the first repository created for the Microverse**
+
+This file was made for review.
+
+
+## Author
+
+👤 **Daniela Reis**
+
+
+## Acknowledgments
+
+- Thanks to Microverse
```
- [x] [OPTIONAL] Kindly add the "Contributing", "Show your support", "Acknowledgments", and the "License" sections along with the MIT license.

ShadyShawkat reviewed (changes requested):
## Hi @danidreis, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

![Image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Required Changes ♻️
- [x] Kindly make sure that the pull request has a descriptive title to it. You can edit the PR's title through the "Edit" button on the top right side of the page.
- [x] Please make sure to give your PR a meaningful description that contains a summarization of the features implemented in the PR. This helps make your PR look more professional. You can edit the PR's description through the three-dot icon on the top right side of the PR's description and then click "Edit".
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

danidreis committed:
Update after first review

Added "social link", "Setup", "Contributing", "Show your support" and the "License" sections.

danidreis committed:
Removed an empty line

danidreis commented:
> ## Hi @danidreis,
> Good job so far! 👍🏻 There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ![Image](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Required Changes ♻️
> * [x]  Kindly make sure that the pull request has a descriptive title to it. You can edit the PR's title through the "Edit" button on the top right side of the page.
> * [x]  Please make sure to give your PR a meaningful description that contains a summarization of the features implemented in the PR. This helps make your PR look more professional. You can edit the PR's description through the three-dot icon on the top right side of the PR's description and then click "Edit".
>   _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._


Thank you

danidreis commented:
Thank you for you kind review!

nicupop729 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+test.md
```
[OPTIONAL FOR THIS PROJECT ONLY]

- [ ] From now on, in all projects we build, we need to have set in the `.gitignore` file the `node_modules/` folder. It is recommended to add it here too.

nicupop729 commented on README.md:
```diff
@@ -1 +1,49 @@
+
+**This is the first repository created for the Microverse**
+
+This file was made for review.
```
[OPTIONAL]

Let's say for this Hello World project it's acceptable (but not recommended) to not have all the sections of the README file completed, but from now on, on each project, we always have to complete all the README sections.

nicupop729 reviewed (approved):
# Status: Project Approved :1st_place_medal: 

Hi @danidreis, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing the requested changes. :confetti_ball: :confetti_ball: 


### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Linters are passing :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
